src/pages/Categories.js

```diff
@@ -0,0 +1,7 @@
+import React from 'react';
+
+export default function Categories() {
+  return (
+    <h1 className="Notification">UNDER DEVELOPMENT!</h1>
```

Comment:
Great job @Kelvin-Ben 🤝

- [ ] [Optional] As stated in the project requirements, kindly make sure this `<h1>` is a button.